src/Modules/Finance/Header/TargetSum/TargetSum.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+
+import styles from '../Form.module.css';
+
+interface ITargetSumProps {
+	value: string | number,
+	buttonValue: string,
+	clicked: (event: React.MouseEvent<HTMLButtonElement>) => void,
+	changed: (event: React.ChangeEvent<HTMLInputElement>) => void,
+	confirmed: boolean,
+	reset: (event: React.MouseEvent<HTMLButtonElement>) => void,
+	corrected: (event: React.MouseEvent<HTMLButtonElement>) => void,
+	touched: boolean
+}
+
+export const TargetSum: React.FC<ITargetSumProps> = props => {

```

Comment:
Please wrap component in `React.memo`.
https://reactjs.org/docs/react-api.html#reactmemo

Reply:
Done